allennlp/data/semparse/knowledge_graphs/table_question_knowledge_graph.py

```diff
@@ -250,3 +285,40 @@ def _get_numbers_from_tokens(tokens: List[Token]) -> List[Tuple[str, str]]:
                         num_zeros += 1
                     numbers.append((str(int(number + 10 ** num_zeros)), token_text))
         return numbers
+
+    cell_part_regex = re.compile(r',\s|\n|/')
+    @classmethod
+    def _get_cell_parts(cls, cell_text: str) -> List[Tuple[str, str]]:
+        """
+        Splits a cell into parts and returns the parts of the cell.  We return a list of
+        ``(entity_name, entity_text)``, where ``entity_name`` is ``fb:part.[something]``, and
+        ``entity_text`` is the text of the cell corresponding to that part.  For many cells, there
+        is only one "part", and we return a list of length one.
+
+        Note that you shouldn't call this on every cell in the table; SEMPRE decides to make these
+        splits only when at least one of the cells in a column looks "splitable".  Only if you're
+        splitting a column should you use this function.
+        """
+        parts = []
+        for part_text in cls.cell_part_regex.split(cell_text):
+            part_text = part_text.strip()
+            part_entity = f'fb:part.{cls._normalize_string(part_text)}'
+            parts.append((part_entity, part_text))
+        return parts
+
+    @classmethod
+    def _should_split_column(cls, column_cells: List[str]) -> bool:
```

Comment:
May be call this method `_should_split_column_cells`?

Reply:
Good point, done.